bash_utils/config.sh

```diff
@@ -21,18 +20,28 @@ function set_env {
             export $(cat $envfile | sed 's/#.*//g' | xargs)
         fi
     done
+    # TODO this is a terrible spot for this, docker setup would make more sense but then .env must be loaded
+    mc config host add spaces $AWS_S3_ENDPOINT $AWS_ACCESS_KEY_ID $AWS_SECRET_ACCESS_KEY --api S3v4
```

Comment:
i feel you with this not feeling great, but seems like a good enough approach since `spaces` get used kind of like an envar and (like you noted) repeatedly loading `.env` later hurts more

Reply:
Main thing here is that it re-adds the space when this is called each time. But this works fine, just redundant. If we keep doing some work to make the dev containers more optimized can do that then - I might make an issue for this just so I don't forget